### DISCUSSION TITLE
location of ca.crt

### DISCUSSION THREAD

**Discussion by jaepetto** [rancher-sandbox/rancher-desktop#5941] [Q&A] [answered]
Hi,

I'm trying to implement a custom admission controller. In order to do that, I need to use the ca.crt file from rancher-desktop.
I followed the steps described here (https://slack-archive.rancher.com/t/12905145/where-do-i-find-the-kubernetes-cluster-certificate-authority)

I also tried to find the certificate on the lima VM (`find / -name ca.rt`), but could not find it.

Does anyone have a pointer?

Best,

Emmanuel

**Comment by jandubois** [comment, marked as answer]
All Kubernetes certs are under `/var/lib/rancher/k3s/server/tls`.

**Reply by jaepetto** [reply]
Thanks @jandubois
